Tests/Brokerages/Bitfinex/BitfinexBrokerageTests.cs

```diff
@@ -52,8 +57,8 @@ protected override IBrokerage CreateBrokerage(IOrderProvider orderProvider, ISec
             return new BitfinexBrokerage(
                     Config.Get("bitfinex-url", "wss://api.bitfinex.com/ws"),
                     Config.Get("bitfinex-rest", "https://api.bitfinex.com"),
-                    Config.Get("bitfinex-api-key"),
-                    Config.Get("bitfinex-api-secret"),
+                    PublicKey,
+                    PrivateKey,
```

Comment:
Why have the credentials as `protected` members (hardcoded in the test class) instead of reading them from the `config.json` file in the tests `bin/Debug` folder?
I believe all other brokerage tests are reading credentials from config